modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts

```diff
@@ -445,6 +445,10 @@ export interface MPCTx {
     commonKeychain?: string;
     lastScanIndex?: number;
   };
+  // the amount recovered using WRW
+  recoveryAmount?: string;
+  // the transaction signature used for broadcasting the transaction
+  signature?: string;
```

Comment:
How is this different from `serializedTx`? Is the idea to deprecate that field? 
If you look at the sol recovery implementation, the `serializedTx` field is used to return the signed tx ready for broadcast

Reply:
To broadcast a Sui transaction, we need both `serializedTx` and `signature`.